Chefs/App.xaml.host.cs

```diff
@@ -19,24 +21,8 @@ private void ConfigureAppBuilder(IApplicationBuilder builder)
 						custom
 							.Login((sp, dispatcher, credentials, cancellationToken) =>
 							{
-								// Check for username to simulate credential processing
-								if (!(credentials?.TryGetValue("Username", out var username) ??
-									  false && !string.IsNullOrEmpty(username)))
-
-								{
-									return ValueTask.FromResult<IDictionary<string, string>?>(null);
-								}
-
-								// Simulate successful authentication by creating a dummy token dictionary
-								var tokenDictionary = new Dictionary<string, string>
-								{
-									{ TokenCacheExtensions.AccessTokenKey, "SampleToken" },
-									{ TokenCacheExtensions.RefreshTokenKey, "RefreshToken" },
-									{ "Expiry", DateTime.Now.AddMinutes(5).ToString("g") } // Set token expiry
-								};
-								return ValueTask.FromResult<IDictionary<string, string>?>(tokenDictionary);
-
-
+								var tokens = ProcessCredentials(credentials);
+								return ValueTask.FromResult(tokens);
```

Comment:
```suggestion
								return ProcessCredentials(credentials);
```
And we can make ProcessCredentials be async and return a `ValueTask<IDictionary<string, string>?>`

We could also switch from block body to a single `.Login(....) => ProcessCredentials(credentials)`